docs/modules/ROOT/pages/cops_rspec_factorybot.adoc

```diff
@@ -171,6 +172,7 @@ create_list :user, 3
 ----
 # bad
 create_list :user, 3
+[create(:user), create(:user), create(:user)]
 
 # good
 3.times { create :user }
```

Comment:
Now when I look at this, it appears that `3.times { ... }` is not a valid replacement for e.g.:
```diff
- users = create_list(:user, 3)
+ users = 3.times { create(:user) }
```
but `user = 3.times.map { create(:user) }` is.

What do you think?

Reply:
Oh, I had not paid attention to this before, but you are right.

Even without this Pull Request, it might be better to always use `times.map` instead for safety when replacing with n_times style. In that case, it would be better to have a rule that `3.times.map { create(:user) }` is replaced with `create_list(:user, 3)` on create_list style side.

Reply:
Even if we change so as in the Pull Request above, the autocorrection from `3.times { create(:user) }` to `create_list(:user, 3)` changes the return value as you commented, so we may want to mark this cop as `SafeAutoCorrect: false`.

Reply:
Fair enough.

For `n_times` style, those cases are unsafe to correct ([origin](https://github.com/rubocop/rubocop-rspec/pull/1469/files#r1016897969)):
If it's inside a let:
```ruby
let(:three_models) { 3.times { create(...) }
```
or is assigned somewhere:
```ruby
three_models = 3.times { create(...) }
```
or used as an argument:
```ruby
create(:user, articles: 3.times { create(:book) })
```
`.map` is certainly essential.

There's an uncertainty for the [dynamic `SafeAutoCorrect` in RuboCop](https://github.com/rubocop/rubocop/issues/11092).

Wondering if your other PR with adding `.map` could reliably fix those cases.